simtools/data_model/validate_data.py

```diff
@@ -55,11 +55,15 @@ def validate_and_transform(self):
 
         """
 
-        self.validate_data_file()
+        if self.data_file_name:
+            self.validate_data_file()
```

Comment:
same as before here. Not sure self.data_file_name is a flag. Also codecov complains exactly in this line that it is not tested

Reply:
Same answer as before (and not related that I've missed this in the tests).